source/_components/switcher_kis.markdown

```diff
@@ -0,0 +1,217 @@
+---
+layout: page
+title: "Switcher"
+description: "Controlling your Switcher V2 Water Heater."
+date: 2018-02-10 18:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: switcher_boiler.png
+ha_category:
+  - Switch
+  - Sensor
+redirect_from:
```

Comment:
```suggestion
```

Because this component is new, there are no existing links on the website, so you can omit this.